common/taskana-common/src/main/resources/sql/postgres/taskana_schema_update_5.10.0_to_6.3.0_postgres.sql

```diff
@@ -0,0 +1,13 @@
+-- this script updates the TASKANA database schema from version 5.1.0 to version 5.2.0.
```

Comment:
Can you change the comment with the correct version?

Reply:
fixed

Reply:
Still a typo in 5.1.0 :)